internal/client/client.go

```diff
@@ -253,6 +253,10 @@ func (a *AuthApiClient) CreateClusterSpec(cmd *cobra.Command, regionInfoList []m
 		faultTolerance, _ := cmd.Flags().GetString("fault-tolerance")
 		clusterInfo.SetFaultTolerance(ybmclient.ClusterFaultTolerance(faultTolerance))
 	}
+	if cmd.Flags().Changed("enterprise-security") {
```

Comment:
should this be hidden under CLI feature flags

Reply:
@shinji62 do we have something in ymb-cli which can be used as feature flags? This feature is not released yet. What happens to docs in that case? can that be hidden as well?

@kc-yb CLI is dependent on ybm deployments, so until we turn on flags there, this too will be behind the flag.

Reply:
@harshitha-yb added some FF based CLI commands. Can you please sync up with her?
The CLI has its own set of FFs

Reply:
https://github.com/yugabyte/ybm-cli/blob/9446d033fbc1ffa20c260a0c2755ddd48fd58fe6/cmd/util/feature_flags.go#L33 you can refer to this @aquatiko